doc/sphinx-guides/source/installation/config.rst

```diff
@@ -185,17 +185,18 @@ There are three valid configurations or modes for authenticating users to Datave
 
```

Comment:
Do you think it might be nice to add some `+++` level headers so that the table of contents at the top looks like this:

<img width="335" alt="Screen Shot 2019-08-21 at 6 15 45 AM" src="https://user-images.githubusercontent.com/21006/63423917-4dadca00-c3db-11e9-8556-458b549aa9cc.png">

Here's a suggestion of where to add the them:

```
 - Local only (also known as "builtin" or "Username/Email").
 - Both local and remote (Shibboleth and/or OAuth).
 - Remote (Shibboleth and/or OAuth) only.
 
+Local Only Auth
++++++++++++++++
+
 Out of the box, Dataverse is configured in "local only" mode. The "dataverseAdm
 
+Both Local and Remote Auth
+++++++++++++++++++++++++++
+
 The ``authenticationproviderrow`` database table controls which "authentication
 
 Adding and enabling a second authentication provider (``Add Authentication Prov
 
 To configure Shibboleth see the :doc:`shibboleth` section and to configure OAut
 
+Remote Only Auth
+++++++++++++++++
+
 As for the "Remote only" authentication mode, it means that:
```

Reply:
Agreed